File: Selenium/Project/crm_activity1.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
#driver=webdriver.Chrome()
with webdriver.Firefox() as driver: 
 
    # Navigate to the URL
 driver.get("http://alchemy.hguy.co/crm")
 # get the title of the page and check the title matches "SuiteCRM"
 if(driver.title=="SuiteCRM"):
        print("Page title is: ", driver.title)
 else:
     print("Wrong page")
     

File: Selenium/Project/CRM_Activity1.java
package crm_project;

import org.testng.annotations.Test;
import org.testng.annotations.BeforeClass;

import static org.testng.Assert.assertEquals;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.annotations.AfterClass;

public class CRM_Activity1 {
 
//declare WebDriver 
		WebDriver driver;
	  
	  
	  @BeforeClass
	  public void setUp() {
		  //initialize webdriver
		  driver = new ChromeDriver();
		  //open the page
		  driver.get("http://alchemy.hguy.co/crm");
	  }
	 
	  @Test
	  public void pageTitleTest() {
		  //get the page title and assert
		
		  assertEquals(driver.getTitle(), "SuiteCRM");
	  }
	 

	  @AfterClass
	  public void tearDown() {
		  //quit the browser
		  driver.quit();
	  }

}
